_posts/2025-02-04-data-wants-to-be-free.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: post
+title: "Data wants to be free: fast data exchange with Apache Arrow"
+description: ""
+date: "2025-02-04 00:00:00"
```

Comment:
We should change the date here (and in the filename) to whatever day this actually posts.

Reply:
I set the date to the 28th